lib/lighthouse/auth/client_credentials/jwt_generator.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
Does lighthouse not already have code to do all this stuff for you? How are others using this? Or are you/we the first?

Reply:
I took some inspiration from `/lib/lighthouse/veterans_health`, but in general I/we am/are one of the first to implement this. I really wanted to use an oauth gem so I didn't have to manage this stuff, but from what I've been able to tell lighthouse's implementation of CCG is non-standard which leads us to where we are now

Reply:
To add context, I don't think that there were many Lighthouse APIs that used CCG as their authentication method in the past, a lot of the older Lighthouse APIs use API keys instead which don't have the same level of effort needed to use